lib/selection/selection.dart

```diff
@@ -37,7 +37,7 @@ typedef E SelectionCallback<E>(datum, int index, Element element);
 typedef E SelectionValueAccessor<E>(datum, int index);
 
 /** Create a ChartedCallback that always returns [val] */
-SelectionCallback toCallback(val) => (datum, index, element) => val;
+SelectionCallback/*<T>*/ toCallback/*<T>*/(/*=T*/ val) => (datum, index, element) => val;
```

Comment:
I just needed a way to tell the analyzer that the return type of this method is a `SelectionCallback<T>` where `T` is the type inferred for `val`